src/mysql_conn.erl

```diff
@@ -851,18 +855,25 @@ demonitor_processes([{_FromPid, MRef}|T], Count) ->
     erlang:demonitor(MRef),
     demonitor_processes(T, Count - 1).
 
-format_status(normal, [_PDict, State]) ->
-	{data, [{"State", State}]};
-format_status(terminate, [_PDict, State=#state{ssl_opts=undefined}]) ->
-	{data, [{"State", State#state{password = hidden}}]};
-format_status(terminate, [_PDict, State=#state{ssl_opts=SSLOpts}]) ->
-	SSLOpts1 = lists:map(
-		fun
-			({cert, _}) -> {cert, hidden};
-			({key, _}) -> {key, hidden};
-			({cacerts, _}) -> {cacerts, hidden};
-			(Other) -> Other
-		end,
-		SSLOpts
-        ),
-	{data, [{"State", State#state{password = hidden, ssl_opts=SSLOpts1}}]}.
+format_status(Status = #{state := State=#state{ssl_opts=SSLOpts}}) ->
+    Status#{state := State#state{password = hidden,
+				 ssl_opts=obfuscate_ssl_options(SSLOpts)}}.
+
+-if(?OTP_RELEASE < 25).
+format_status(_, [_PDict, State=#state{ssl_opts=SSLOpts}]) ->
+    {data, [{"State", State#state{password = hidden,
+				  ssl_opts=obfuscate_ssl_options(SSLOpts)}}]}.
```

Comment:
I'm not sure it's consistent in this repo, but I think the idea is to always to have spaces around `=`.

```suggestion
format_status(Status = #{state := State = #state{ssl_opts = SSLOpts}}) ->
    Status#{state := State#state{password = hidden,
				 ssl_opts = obfuscate_ssl_options(SSLOpts)}}.

-if(?OTP_RELEASE < 25).
format_status(_, [_PDict, State = #state{ssl_opts = SSLOpts}]) ->
    {data, [{"State", State#state{password = hidden,
				  ssl_opts = obfuscate_ssl_options(SSLOpts)}}]}.
```

Reply:
Applied and squashed :)